test/e2e/knative_eventing_test.go

```diff
@@ -34,43 +35,35 @@ func TestKnativeEventing(t *testing.T) {
 	})
 
 	t.Run("deploy knativeeventing cr and wait for it to be ready", func(t *testing.T) {
-		if _, err := v1a1test.WithKnativeEventingReady(caCtx, knativeEventing, knativeEventing); err != nil {
+		if _, err := v1a1test.WithKnativeEventingReady(caCtx, eventingName, eventingNamespace); err != nil {
 			t.Fatal("Failed to deploy KnativeEventing", err)
 		}
 	})
 
 	t.Run("verify correct deployment shape", func(t *testing.T) {
 		for i := range knativeControlPlaneDeploymentNames {
 			deploymentName := knativeControlPlaneDeploymentNames[i]
-			if _, err := test.WithDeploymentReady(caCtx, deploymentName, knativeEventing); err != nil {
+			if _, err := test.WithDeploymentReady(caCtx, deploymentName, eventingNamespace); err != nil {
 				t.Fatalf("Deployment %s is not ready: %v", deploymentName, err)
 			}
 		}
-
-		if err := test.WithDeploymentCount(caCtx, knativeEventing, len(knativeControlPlaneDeploymentNames)); err != nil {
```

Comment:
Why delete this?

Reply:
Well, mainly because eventually there will be other deployments in that ns besides eventing, but even as it stands, I couldn't see how this test offered any benefit over the loop immediately preceding it.

Reply:
Eh, good one :joy: